input/fsh/Instance-Patient.fsh

```diff
@@ -6,6 +6,20 @@ Instance: patient-example-1
 InstanceOf: Patient
 Usage: #example
 Description: "Example of the representation of Patient."
+* extension[0].url = $us-core-race
```

Comment:
@bwalsh, for the US Core Race and Ethnicity extensions, those slicing indexes are required (the first one is always `url`, the second for `ombCategory`, and the third for `detailed`). Any suggestions for docstring? 

Since we are not developing these extensions, but simply including in our examples, the implementation instruction may not be required though?

Reply:
Good point.  Adding more documentation here would be extraneous. ETL author could refer to extension doc. 
 e.g. http://hl7.org/fhir/us/core/StructureDefinition/us-core-race